src/c_kzg_4844.c

```diff
@@ -254,6 +254,20 @@ static bool fr_is_one(const fr_t *p) {
     return a[0] == 1 && a[1] == 0 && a[2] == 0 && a[3] == 0;
 }
 
+/**
+ * Test whether the operand is zero in the finite field.
+ *
+ * @param[in] p The field element to be checked
+ *
+ * @retval true  The element is zero
+ * @retval false The element is not zero
+ */
+static bool fr_is_zero(const fr_t *p) {
+    uint64_t a[4];
+    blst_uint64_from_fr(a, p);
+    return a[0] == 0 && a[1] == 0 && a[2] == 0 && a[3] == 0;
```

Comment:
Why not just compare it with `FR_ZERO`? like
```c
return fr_equal(p, &FR_ZERO);
```

Reply:
From another point of view, this function is not necessary at all, since you can use `fr_equal` with `FR_ZERO`, but I'm fine with either approach.

Reply:
The idea was to avoid a second `blst_uint64_from_fr()` which is a moderately expensive operation.

Reply:
Thank makes sense